x-pack/plugins/security/server/lib/authorization/service.js

```diff
@@ -23,11 +24,13 @@ export function createAuthorizationService(server, xpackInfoFeature) {
     shieldClient,
     xpackInfoFeature,
   );
+  const privileges = buildPrivilegeMap(savedObjectTypes, actions, features);
```

Comment:
What do you think about freezing this, since it's part of a "public" service now?

Reply:
Having it here is actually going to cause us problems because of the way that the Kibana "lifecycle" works and it's being frozed via the "deep freeze" that we do on the authorization service currently.

The issue with initializing this here is that not all of the other plugins are guaranteed to have ran their init yet, so if we determine this here and then use it elsewhere it might be missing apps... I'm gonna play around with making this either "getPrivileges" to overcome this limitation, or instantiating it where we need it like we used to be.